presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -140,6 +142,7 @@ public Statement rewrite(
         return (Statement) new Visitor(metadata, parser, session, parameters, accessControl).process(node, null);
     }
 
+    @SuppressWarnings("BooleanMethodIsAlwaysInverted")
```

Comment:
Don't suppress on the class level, unless the suppression applies to the whole class.